test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_transfer.py

```diff
@@ -119,9 +119,6 @@ def test_insufficient_balance_for_fee(spec, state):
     state.balances[sender_index] = spec.MAX_EFFECTIVE_BALANCE
     transfer = get_valid_transfer(spec, state, sender_index=sender_index, amount=0, fee=1, signed=True)
 
-    # un-activate so validator can transfer
-    state.validators[transfer.sender].activation_epoch = spec.FAR_FUTURE_EPOCH
```

Comment:
Why remove this? An active validator can't transfer so this will fail regardless of balance.
Seems like it just needs to be `activation_eligibililty_epoch`

Reply:
you are right on this one. But see ditto comment. That one is correct.

Reply:
fixed in rebase